engine/interpreter-dsl-test/src/test/java/org/enso/interpreter/dsl/test/InliningBuiltinsTest.java

```diff
@@ -8,141 +8,131 @@
-          ctx,
-          () -> {
-            var fn = InliningBuiltinsInMethodGen.makeFunction(null);
-            if (fn.getCallTarget().getRootNode() instanceof InlineableNode.Root root) {
-              var call = root.createInlineableNode();
-              var clazz = call.getClass();
-              assertEquals("InlineableNode", clazz.getSuperclass().getSimpleName());
-              assertEquals(
-                  "org.enso.interpreter.node.InlineableNode$Root",
-                  clazz.getEnclosingClass().getInterfaces()[0].getName());
+    ctxRule.executeInContext(
+        () -> {
+          var fn = InliningBuiltinsInMethodGen.makeFunction(null);
+          if (fn.getCallTarget().getRootNode() instanceof InlineableNode.Root root) {
+            var call = root.createInlineableNode();
+            var clazz = call.getClass();
+            assertEquals("InlineableNode", clazz.getSuperclass().getSimpleName());
+            assertEquals(
+                "org.enso.interpreter.node.InlineableNode$Root",
+                clazz.getEnclosingClass().getInterfaces()[0].getName());
 
-              var res =
-                  WithFrame.invoke(
-                      (frame) -> {
-                        return call.call(
-                            frame,
-                            Function.ArgumentsHelper.buildArguments(
-                                null, new Object[] {null, 5L, 7L}));
-                      });
-              assertEquals(12L, res);
-            } else {
-              fail("It is inlineable: " + fn.getCallTarget().getRootNode());
-            }
-            return null;
-          });
-    }
+            var res =
+                WithFrame.invoke(
+                    (frame) -> {
+                      return call.call(
+                          frame,
+                          Function.ArgumentsHelper.buildArguments(
+                              null, new Object[] {null, 5L, 7L}));
+                    });
+            assertEquals(12L, res);
+          } else {
+            fail("It is inlineable: " + fn.getCallTarget().getRootNode());
+          }
+          return null;
+        });
   }
 
   /**
    * @see InliningBuiltinsOutNode#execute(com.oracle.truffle.api.frame.VirtualFrame, long, long)
    */
   @Test
   public void executeWithVirtualFrame() {
-    try (var ctx = ContextUtils.createDefaultContext()) {
-      ContextUtils.executeInContext(
-          ctx,
-          () -> {
-            var fn = InliningBuiltinsOutMethodGen.makeFunction(null);
-            if (fn.getCallTarget().getRootNode() instanceof InlineableNode.Root root) {
-              fail("The node isn't inlineable: " + fn.getCallTarget().getRootNode());
-            } else {
-              var call = DirectCallNode.create(fn.getCallTarget());
-              var clazz = call.getClass().getSuperclass();
-              assertEquals("com.oracle.truffle.api.nodes.DirectCallNode", clazz.getName());
+    ctxRule.executeInContext(
```

Comment:
Why not `executeInContext` by default? `TestRule` can wrap the execution of a `Statement` by entering and leaving the context.

Reply:
Done in 08acc4c1480044eeae13acf2581874d34090c0b2